validator/gke_cluster_location_test.rego

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GKEClusterLocationConstraintV1
+
+all_violations[violation] {
+	resource := data.test.fixtures.gke_cluster_location.assets[_]
+	constraint := data.test.fixtures.gke_cluster_location.constraints
+
+	issues := deny with input.asset as resource
+		 with input.constraint as constraint
+
+	violation := issues[_]
+}
+
+test_gke_cluster_location_violations_basic {
+	count(all_violations) == 2
+	violation := all_violations[_]
+	contains(violation.details.resource, "joe-clust")
```

Comment:
Could you check the exact set of resources in violation for future-proofing your test?

something like [this](https://github.com/forseti-security/policy-library/blob/master/validator/appengine_location_test.rego#L50) would be good.